src/main/java/com/taf/business/api/ApiTestContext.java

```diff
@@ -0,0 +1,21 @@
+package com.taf.business.api;
+
+import io.restassured.response.ValidatableResponse;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Getter
+public class ApiTestContext {
+    ApiClient apiClient = new ApiClient();
+    Request request = new Request();
+    @Setter
+    ValidatableResponse response;
+    @Setter
+    Map<String, String> variables;
+
+    public void addVariable(String name, String value) {
+        variables.put(name, value);
```

Comment:
if you won't set map and use this method you'll receive NPE. I propose to check if variables exists first before adding data

Reply:
Fixed after code refactoring.